### PR TITLE
Autoconfiguration

### PR DESCRIPTION
**Pull request by blaugueux**
Allows EventSubscriber to be autoconfigured with symfony ~3.3.

### CODE REVIEW

**Comment by goetas** [comment]
Any news on this?

File: DependencyInjection/JMSSerializerExtension.php [hunk 1]
```diff
@@ -30,6 +30,10 @@ class JMSSerializerExtension extends ConfigurableExtension
 {
     public function loadInternal(array $config, ContainerBuilder $container)
     {
+        $container
```
Comment (juntereiner) [inline DependencyInjection/JMSSerializerExtension.php]: `if (method_exists($container, 'registerForAutoconfiguration')) {
            $container
                ->registerForAutoconfiguration(EventSubscriberInterface::class)
                ->addTag('jms_serializer.event_subscriber');
        }`

Should fix the issue with sf  < 3.3

Comment (blaugueux) [reply DependencyInjection/JMSSerializerExtension.php]: Thx @juntereiner for your idea

**Comment by goetas** [comment]
any way to add a test for it?

**Comment by juntereiner** [comment]
I wasn't sure @blaugueux would respond to my comment so quickly, so I made an another merge request #633 
You can either complete this one or look mine if that's ok.

**Comment by goetas** [comment]
solved by https://github.com/schmittjoh/JMSSerializerBundle/pull/633

**Pull request closed without merging**